desktop-widgets/groupedlineedit.cpp

```diff
@@ -175,7 +175,11 @@ void GroupedLineEdit::paintEvent(QPaintEvent *e)
 	painter.setRenderHint(QPainter::Antialiasing, true);
 	painter.fillRect(0, 0, viewport()->width(), viewport()->height(), palette().base());
 
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
```

Comment:
You shouldn't need this if you `#include <qvector.h>` in addition to `qlist.h`. See https://github.com/qt/qtbase/blob/dev/src/corelib/tools/qvector.h#L48-L53

Reply:
ok, let me try that again. I couldn't make it work and so I ended up with this solution.